.gitignore

```diff
@@ -462,3 +462,10 @@ pkg/apprclient/openapi/git_push.sh
 !test/**testdata/**
 
 /coverage.out
+
+# Test artifacts generated by e2e
+test/e2e/bundle.Dockerfile
+test/e2e/downloaded/**
+test/e2e/opm-*
+test/e2e/bundle_tmp*
+test/e2e/index_tmp*
```

Comment:
In general I think the tests do clean themselves up assuming they are allowed to run to completion. If you're debugging and stop execution, those artifacts get left around. Then they show up as possible items to commit, and the only way to prevent accidental checkin is to add them to .gitignore.